lib/utils/check.ts

```diff
@@ -0,0 +1,64 @@
+import isEmpty from 'lodash/isEmpty';
+import isPlainObject from 'lodash/isPlainObject';
+import type { DataType } from '../..';
+import { getOperators } from './format';
+// eslint-disable-next-line import/order
```

Comment:
Is there a reason that we don't want the requested order?

Reply:
Will not duplicate this comment over the other files since I think the reason will be the same in each case

Reply:
Ah, later on you mention; "-- caused by temporarily mixing require with import". Maybe copy that line over to all files?

Reply:
Fixed. Well, documented really :)